### PR TITLE
added adhoc instance docs

### PR DESCRIPTION
**Pull request by mjksmith**
added adhoc instance docs

### CODE REVIEW

File: docs/source/adhoc_instances.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+A sample use case where one needs to ssh onto an adhoc batch machine and run
+the adhoc instance ``example_instance`` for the service ``example_service``
+would use the command:
+
+  ``paasta local-run --pull -s example_service -i example_instance``
+
+The 'interactive' instance
+--------------------------
+
+Running ``paasta local-run`` without specifying the ``--instance`` flag creates
```
Comment (solarkennedy) [inline docs/source/adhoc_instances.rst]: How about "launches" instead of "creates"?

File: docs/source/adhoc_instances.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Running an adhoc instance
+=========================
+
+Adhoc instances can be run using ``paasta local-run`` like any other instance.
+A sample use case where one needs to ssh onto an adhoc batch machine and run
+the adhoc instance ``example_instance`` for the service ``example_service``
+would use the command:
+
+  ``paasta local-run --pull -s example_service -i example_instance``
```
Comment (solarkennedy) [inline docs/source/adhoc_instances.rst]: Use long args for examples always

File: docs/source/adhoc_instances.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
+=================================
+Running Adhoc Instances in PaaSTA
+=================================
+
+PaaSTA allows users to pre-define and run adhoc instances. These instances can
+be used to run batch jobs, test suites, etc.
```
Comment (solarkennedy) [inline docs/source/adhoc_instances.rst]: Can you throw on the word "one-off" for people who have heard of heroku:
https://devcenter.heroku.com/articles/one-off-dynos

**Comment by mjksmith** [comment]
Crap I've messed this PR up

**Pull request merged**